src/java/edu/ucsd/library/dams/api/FedoraAPIServlet.java

```diff
@@ -1029,12 +1044,17 @@ else if ( suppressDiscovery && discover )
 			// suppress discovery of hidden objects
 			accessGroup = roleAdmin;
 		}
-		else if (    copyright.equalsIgnoreCase("public domain") ||
-				( copyright.equalsIgnoreCase("under copyright") &&
-				rightsHolder.equalsIgnoreCase(localCopyright) )      )
+		else if ( copyright.equalsIgnoreCase("public domain")
+					|| ( copyright.equalsIgnoreCase("under copyright") && rightsHolder.equalsIgnoreCase(localCopyright) )
```

Comment:
should we make the `"under copyright"` string a static variable similar to `VISIBILITY_LOCAL` that you introduced?